mw-webapp/src/logic/wayPage/WayStatistic.tsx

```diff
@@ -14,6 +14,11 @@ interface WayStatisticProps {
    * Reports of specific way
    */
   dayReports: DayReport[];
+
+  /**
+   * Date of way created
+   */
+  wayCreated: Date;
```

Comment:
is it convinietnNaming? didn't we use 'createdAt'?

Reply:
renamed